sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -227,6 +225,7 @@ def test_big_query_standard_sql_kms_key_native(self):
         'on_success_matcher': all_of(*pipeline_verifiers),
         'kms_key': kms_key,
         'native': True,
+        'experiments': 'use_dataflow_bq_sink',
```

Comment:
Can we just change this sink to use the new sink ?

Reply:
I'm trying to preserve the same test coverage after the change, and remove BQSink tests later on. Would htis be fine?

Reply:
I think we should either change tests to use the default sink (which will be used by most of the users going forward) or run tests for both.